README.md

```diff
@@ -12,11 +12,11 @@ This tool generates architecture models for ETISS from CoreDSL source files or a
 - Install Python dependencies: `pip install -r requirements.txt`
 
 ## Architecture
-M2-ISA-R consists of 3 components, two of which are exchangable for different needs:
+M2-ISA-R consists of 3 components, two of which are exchangeable for different needs:
```

Comment:
This is a matter of British vs. American English. I tend to stick to AE, so this can stay